create_framework/http_kernel_httpkernelinterface.rst

```diff
@@ -46,8 +46,8 @@ Update your framework so that it implements this interface::
         }
     }
 
-Even if this change looks not too complex, it brings us a lot! Let's talk about one of
-the most impressive one: transparent :doc:`HTTP caching </http_cache>` support.
+Even if this change doesn't look like much, it brings us a lot! Let's talk about one of
+the most impressive ones: transparent :doc:`HTTP caching </http_cache>` support.
```

Comment:
To me, it seems like "one" is correct here, given only 1 things is mentioned after it?

Reply:
To me, "ones" was correct because we talk about 1 feature among several impressive ones ( = features).

Reply:
> Let's talk about one of the most impressive ones

The first "`one`" is referring to an item in the set of "impressive ones".

I am not 100% but I think @Ca-Jou is correct here.

Reply:
Oh sorry, I read this way too quick. (I was thinking it said "Let's talk about the most impressive one:")